arch/x86/kvm/pkvm/mmu.c

```diff
@@ -11,8 +11,9 @@
 /* FIXME: Create a generic layer to abstract the VMX specific functions  */
 #include "vmx/mmu.h"
 
-static struct pkvm_pgtable hyp_mmu;
+static struct pkvm_pgtable host_mmu;
```

Comment:
nit: move host_mmu definition after hyp_mmu and hyp_mmu_pool (just because host MMU is initialized later, and the entire code below for host MMU is after the code for hypervisor MMU)?